### PR TITLE
feat/TDP-3291 nav components review

### PR DESCRIPTION
**Pull request by douglasmik**
Update of documentation and optimisation of Newskit Nav Components
https://nidigitalsolutions.jira.com/browse/TDP-3291

### CODE REVIEW

File: packages/ts-newskit/src/assets/TimesMasthead.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ const TimesMasthead = (props: any) => (
     xmlns="http://www.w3.org/2000/svg"
     {...props}
   >
-    <g clip-path="url(#clip0_1565_54429)">
```
Comment (adamosborne-tnl) [inline packages/ts-newskit/src/assets/TimesMasthead.tsx:12]: i thought this was ok before. `clip-path` is a property, but `<clipPath/>` is an element that you could reference, which you done. Or is this some TS quirk?

File: packages/ts-newskit/src/components/global-nav/index.tsx [hunk 2]
```diff
@@ -4,17 +4,17 @@ import HamburgerMenu from './hamburger-menu/HamburgerMenu';
 import { NavigationData } from './types';
 import { HamburgerMenuContainer } from './styles';
 
-type GlobalNavProps = {
+interface GlobalNavProps {
   isLoggedIn?: boolean;
```
Comment (adamosborne-tnl) [inline packages/ts-newskit/src/components/global-nav/index.tsx:8]: if you have default values, are these no longer conditional?

**Review by adamosborne-tnl — APPROVED** [review]

File: packages/ts-newskit/src/assets/TimesMasthead.tsx (hunk 1 above)
Comment (douglasmik) [reply packages/ts-newskit/src/assets/TimesMasthead.tsx:12]: As we've put it as a JSX component we need to put this to camel case, maintaining it as a property

File: packages/ts-newskit/src/components/masthead/styles.ts [hunk 3]
```diff
@@ -1,25 +1,13 @@
...
-export const Masthead = styled.div`
-  width: 100%;
+export const Masthead = styled(Block)`
   border-bottom: 1px solid #e4e4e4;
   text-align: center;
-  display: block;
+`;
+
+export const MastheadDate = styled(DateTime)`
+  texttransform: 'capitalize';
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/masthead/styles.ts:9]: Does this need a hyphen between? text-transform? probably not just checking :)

Comment (bibicollins1) [inline packages/ts-newskit/src/components/masthead/styles.ts:9]: Is this meant to have a hyphen?

**Review by bibicollins1 — APPROVED** [review]
Amazing work!

**Review by ibrahim-kurhan — APPROVED** [review]

File: packages/ts-newskit/src/components/masthead/styles.ts [hunk 4]
```diff
@@ -1,25 +1,13 @@
...
-export const Masthead = styled.div`
-  width: 100%;
+export const Masthead = styled(Block)`
   border-bottom: 1px solid #e4e4e4;
   text-align: center;
-  display: block;
+`;
+
+export const MastheadDate = styled(DateTime)`
+  text-transform: 'capitalize';
```
[the remark this answers is not in the document]
Comment (douglasmik) [reply packages/ts-newskit/src/components/masthead/styles.ts:9]: It is? Not 'uppercase' but date should have each word 'capitalised';
e.g. **F**riday **F**ebruary 17 2023

[the remark this answers is not in the document]
Comment (bibicollins1) [reply packages/ts-newskit/src/components/masthead/styles.ts:9]: Think Capitalize just does the first letter of each word so - Friday 24th March - Uppercase does all of the characters. (If I'm correct)

**Pull request merged**